src/test/java/com/wikia/webdriver/testcases/seotests/Our404HandlerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class Our404HandlerTest extends NewTestTemplate {
+  @DataProvider
+  private Object[][] testCasesForOur404Handler() {
+    return new Object[][]{
+        {"muppet", "sitemap-index.xml", "sitemap-index.xml"},
+        {"muppet", "robots.txt", "robots.txt"},
+        {"muppet", "Miss_Piggy", "wiki/Miss_Piggy"},
+        {"pl.gta", "Special:Version", "wiki/Specjalna:Wersja"},
+        {"pl.gta", "Special:Wersja", "wiki/Specjalna:Wersja"},
+        {"pl.gta", "Specjalna:Version", "wiki/Specjalna:Wersja"},
+        {"pl.gta", "Specjalna:Wersja", "wiki/Specjalna:Wersja"},
+        {"www", "NoSuchPage", "NoSuchPage"},
+// Current code doesn't redirect to non-existing articles:
+//        {"wowwiki", "NoSuchPage", "wiki/NoSuchPage"},
+//        {"muppet", "NoSuchPage", "wiki/NoSuchPage"},
+        {"muppet", "wiki/Miss_Piggy", "wiki/Miss_Piggy"},
+        {"www", "About", "About"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "About" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)